mozetl/taar/taar_amowhitelist.py

```diff
@@ -0,0 +1,86 @@
+#!/bin/env python
+
+import click
+import json
+import logging
+import logging.config
+from dateutil.parser import parse
+from tempfile import NamedTemporaryFile
+import datetime
+
+from taar_utils import read_from_s3, store_json_to_s3
+
+AMO_DUMP_BUCKET = 'telemetry-parquet'
+AMO_DUMP_PREFIX = 'telemetry-ml/addon_recommender/'
+
+AMO_DUMP_BASE_FILENAME = 'extended_addons_database'
+FILTERED_AMO_BASE_FILENAME = 'whitelist_addons_database'
+
+AMO_DUMP_FILENAME = AMO_DUMP_BASE_FILENAME + '.json'
+FILTERED_AMO_FILENAME = FILTERED_AMO_BASE_FILENAME + '.json'
+
+MIN_RATING = 3.0
+MIN_AGE = 60
+
+logger = logging.getLogger('amo_whitelist')
+
+
+class AMOTransformer:
+    def __init__(self, bucket, prefix, fname, min_rating, min_age):
+        self._s3_bucket = bucket
+        self._s3_prefix = prefix
+        self._s3_fname = fname
+        self._min_rating = min_rating
+        self._min_age = min_age
+
+    def extract(self):
+        with NamedTemporaryFile() as dst_json_file:
```

Comment:
The same can be achieved with something like `json.loads(s3.Object(BUCKET_NAME, S3_KEY).get()['Body'])` without resorting to a temporary file. Just FYI if you want to use that rather than this.

Reply:
Thanks!  I've put that code into the taar_utils.read_from_s3 function now.